### PR TITLE
📖 amp-list: fix amp.dev docs syntax errors

### PR DESCRIPTION
**Pull request by matthiasrohmer**
/cc @sebastianbenz @patrickkettner @CrystalOnScript @robinvanopstal

### CODE REVIEW

File: extensions/amp-list/amp-list.md [hunk 1]
```diff
@@ -578,3 +578,5 @@ The AMP for Email spec disallows the use of the following attributes on the AMP
 ## Validation
 
 See [amp-list rules](https://github.com/ampproject/amphtml/blob/master/extensions/amp-list/validator-amp-list.protoascii) in the AMP validator specification.
+
+[/filter]
```
Comment (caroqliu) [inline extensions/amp-list/amp-list.md:582]: Does this need to be changed both here and in #29550?

Comment (matthiasrohmer) [reply extensions/amp-list/amp-list.md:585]: Nope, I just didn't see Patrick's PR 😁  both will do the trick and unblock the amp.dev's build

Comment (caroqliu) [reply extensions/amp-list/amp-list.md:585]: Oh I see. :) There I recommended removing the opening tag instead of adding the closing tag.

Comment (matthiasrohmer) [reply extensions/amp-list/amp-list.md]: Done!

**Comment by sebastianbenz** [comment]
@caroqliu are the visual diff test failures related to this change?

**Comment by caroqliu** [comment]
> @caroqliu are the visual diff test failures related to this change?

Looks like the tests were failing due to not finishing. I restarted it in Travis, looks good to go now.

**Comment by sebastianbenz** [comment]
Thanks @caroqliu!

**Pull request merged**